utils/utils.go

```diff
@@ -34,6 +35,20 @@ import (
 )
 
 var WarningSign = "\U000026A0"
+var JwtToken = ""
+var APIServerURL string
+
+type GHAuthReq struct {
```

Comment:
I think these login related structs and methods below should go into a separate package in platform and ideally even interfaced.

Reply:
yep can be done.

Reply:
done